content/docs/command-reference/config.md

```diff
@@ -127,8 +127,8 @@ remote. See `dvc remote` for more information.
 A DVC project <abbr>cache</abbr> is the hidden storage (by default located in
 the `.dvc/cache` directory) for files that are tracked by DVC, and their
 different versions. (See `dvc cache` and
-[DVC Files and Directories](/doc/user-guide/dvc-files-and-directories#structure-of-the-cache-directory)
-for more details.) This section contains the following options:
+[DVC Files and Directories](/doc/user-guide/dvc-files-and-directories) for more
```

Comment:
Hmmm, I think this should link to /doc/user-guide/glossary/dvc-cache#structure-of-the-cache-directory now, no?

Reply:
Yep, fixed.